Dockerfile

```diff
@@ -3,15 +3,16 @@ FROM ubuntu:16.04
 ARG userid=1000
 
 RUN mkdir /var/run/sshd
-RUN apt-get update
-RUN apt-get install -y openssh-server sudo
-
-# Required by bitbake
-RUN apt-get install -y cpio iputils-ping
 
+# Install dependencies in one command to avoid potential use of previous cache
+# like explained here: https://stackoverflow.com/a/37727984
+RUN apt-get update \
+    && apt-get install -y openssh-server sudo cpio inetutils-ping locales
+    
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
Why is this line needed? Can't we just add a comment saying "always do update && install as one RUN instruction" ?

Reply:
I added the comment. I have no idea why the `rm` line is there, it was there before too, I didn't want to remove it.

Reply:
Aha, I thought the `rm` line was to avoid issues or something. Because it shows as an addition in the diff.